buddyboss/buddyboss-platform
# Fixed the group visibility translation issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1716 .

### How to test the changes in this Pull Request:

1. try to translate the Private/Public/Hidden using Loco Translate
2. Go to any group.
3. Send an invite to someone
4. check the email and notice the error

### Proof Screenshots or Video

https://www.loom.com/share/79ef8d6cf55a4be08d0d08df06c31791

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
